src/applications/personalization/profile/util/deriveContactInfoProperties.js

```diff
@@ -0,0 +1,94 @@
+import { API_ROUTES, FIELD_TITLES, FIELD_NAMES } from '@@vap-svc/constants';
```

Comment:
Naming things is hard. I like this collection of helpers and such, I'm just not crazy about the name of the file. For example, importing the `phoneNumbers` from 'deriveContactInfoProperties' reads a little weird to me. I don't have a great suggestion for a better name or organization strategy though :(